Handle empty git diff & fix a title typo

sywesk opened the pull request:
- Corrected the CLI title from "SGPT DIF" to "SGPT DIFF"
 - Added an error check to abort when the git diff is empty

malonaz commented on diff/diff.go:
```diff
@@ -100,6 +100,10 @@ func NewCmd(openAIClient *openai.Client, config *configuration.Config) *cobra.Co
 			gitDiff := bytesBuffer.String()
 
+			if len(gitDiff) == 0 {
+				cobra.CheckErr("git diff is empty, aborting.")
```
Ca marche sans passer d erreur?

malonaz replied:
J aurai imagine que y'a un cobra.Fail()?

sywesk replied:
Pour la postérité: il n'y a pas d'alternative à `CheckErr` dans cobra.

sywesk committed:
[diff] - fix: correct command title and handle empty diff scenario

 - Corrected the title of the command from "SGPT DIF" to "SGPT DIFF" for consistency and clarity
 - Added a condition check for empty git diff and abort process if true to prevent further processing or potential errors
